### DISCUSSION TITLE
I've got an animated model that is in the wrong scale

### DISCUSSION THREAD

**Discussion by filahf** [google/model-viewer#1963] [Q&A]
As the title says, the scale is wrong. I've tried to edit the file in blender but the animation will glitch out.

Is there anyway to reach down to the scale functionality of three.js while in web-xr mode?

Thanks in advance

**Comment by filahf** [comment]
I managed to solve it temporarily by modifying goalScale

**Comment by elalish** [comment]
You might want to try the `scale` attribute that was just added in v1.4: https://modelviewer.dev/docs/index.html#entrydocs-scenegraph-attributes-scale

**Reply by Gabby604** [reply]
> You might want to try the `scale` attribute that was just added in v1.4: https://modelviewer.dev/docs/index.html#entrydocs-scenegraph-attributes-scale

Hi !

I tried to add the Scale function but I didn't see any difference in my model. Do I have to add this function before others or it's not important?
Also, what are the correct value ? scale="1.5 1.5 1.5" ? or I have to add x, y, z too?

Thanks !

**Reply by elalish** [reply]
The first way is correct. It's not a function, it's an attribute, so it goes inside the brackets: `<model-viewer scale="1.5 1.5 1.5">`. Scale won't change 3D mode because our camera auto-frames. It'll only make a difference in AR, and only once the object has been properly placed on the floor.
